Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -242,172 +254,220 @@ edgebridgetutorialapp://
+To learn more about the final data format created by Edge Bridge, please refer to the [data format documentation](../data-format.md).
 
-## Data Prep mapping
+#### Trigger rule-based `trackAction` events <!-- omit in toc -->
+Rule-based `trackAction`/`trackState` events are also converted to Edge events by the Edge Bridge extension. Select the **Trigger Consequence** button (**1**) to initiate a rule that generates a `trackAction` event.
 
-<details>
-  <summary> Data Prep background</summary><p>
+<img src="../assets/edge-bridge-tutorial/assurance-validation/ios-app-trigger-rule-button.png" alt="Simulator tracking buttons" width="400"/>
 
-Data Prep is an Adobe Experience Platform service which maps and transforms data to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html).  Data Prep is configured from a Platform enabled [datastream](https://experienceleague.adobe.com/docs/experience-platform/edge/datastreams/overview.html) to map source data from the Edge Bridge mobile extension to the Platform Edge Network.
+> [!TIP]
+> If no `trackAction` event is dispatched upon selecting the **Trigger Consequence** button, verify that your mobile property is [set up with the correct rule](#2-configure-a-rule-to-forward-personal-identifiable-information-pii-events-to-edge-network).
 
-This guide covers how to map data sent from the Edge Bridge within the Data Collection UI.
+Just like the `trackAction`/`trackState` events above, the Edge Bridge extension converts the personal PII `trackAction` event into an Edge event.
 
-For a quick overview of the capabilities of Data Prep, watch the following [video](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html).
+With this, the migration from an existing Analytics implementation to sending data via Edge Network to Analytics, powered by Edge Bridge, is complete!
 
-> **Note**
-> The following documentation provides a comprehensive overview of the Data Prep capabilities:
-> - [Data Prep overview](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html)
-> - [Data Prep mapping functions](https://experienceleague.adobe.com/docs/experience-platform/data-prep/functions.html)
-> - [Handling data formats with Data Prep](https://experienceleague.adobe.com/docs/experience-platform/data-prep/data-handling.html)
->
+## Optional - Data Prep for Data Collection mapping
+Edge Bridge transforms Analytics track events into an Edge Network event format suitable for Analytics. To use this same event data in other Experience Platform applications, it must first be mapped to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html) specification using [Data Prep for Data Collection](https://experienceleague.adobe.com/en/docs/experience-platform/datastreams/data-prep).
```

Comment:
some suggestions:
- suitable for Adobe Analytics
- let's also clarify why this step is optional, eg. below:

Edge Bridge transforms track events into an Edge Network event format suitable for Adobe Analytics, so mapping data to XDM is no longer required for Analytics. However, to use this same event data in other Experience Platform applications, it must first be mapped to ...